cmd/osde2e/cleanup/cmd.go

```diff
@@ -54,54 +59,90 @@ func init() {
 		"",
 		"A specific cluster id to cleanup",
 	)
-
+	flags.BoolVar(
+		&args.iam,
+		"iam",
+		false,
+		"Cleanup iam resources",
+	)
+	flags.IntVar(
+		&args.olderThan,
+		"older-than",
+		1,
+		"Cleanup iam resources older than this number of days",
+	)
+	flags.BoolVar(
+		&args.dryrun,
+		"dry-run",
+		true,
+		"Show dry run log of deleting iam resources",
+	)
```

Comment:
these flags should be named more specifically if they apply to only `iam` (`--iam-dry-run`, `--iam-older-than`)

Reply:
though it doesn't seem either of these are being used